.github/ci/update_readme_dependencies.py

```diff
@@ -103,6 +103,13 @@ def get_capabilities_table(header_path, manifest):
         return table_data
 
 
+def index_regexp(yourlist, yourstring):
+    ide = [i for i, item in enumerate(yourlist) if re.search(yourstring, item)]
+    if len(ide) > 0:
+        return ide[0]
+    return -1
+
```

Comment:
just a suggestion, not necessary to change
```suggestion
def index_regexp(yourlist, yourstring):
    for i, line in enumerate(yourlist):
        if yourstring in line:
            return i
    return -1
```

Reply:
Thank you for suggestion, it looks good :-) Changed.